### DISCUSSION TITLE
Filter out values  in AWS Secrets manager's secrets with TriggerAuthentification

### DISCUSSION THREAD

**Discussion by stepan111** [kedacore/keda#5785] [Q&A / Need Help] [answered]
Hello Colleagues,
First of all, thank for great product !
I've done my KEDA setup on EKS and now I am trying to retrieve secrets stored in AWS Secrets Manager. I've succeed in creating test secret, retrieving it's value and passing as parameter to Scaledjob. 
Secret created like this:
```
aws secretsmanager create-secret \
    --name MyTestSecret \
    --secret-string "connection://string:here"
```
And retrived by TriggerAuthentication
```
    secrets:
    - name: MyTestSecret
      parameter: host

```
Currently my concern is that we creating secrets in slightly other manner (having a json in secret string):
```
aws secretsmanager create-secret \
    --name MyTestSecret \
    --description "My test secret created with the CLI." \
    --secret-string "{\"user\":\"diegor\",\"password\":\"EXAMPLE-PASSWORD\"}"

```
So my question would be if I can filter out some value in such secret string with TriggerAuthentication? For example how it is done with k8s secrets by specifying a key. [Docs reference](https://keda.sh/docs/2.14/concepts/authentication/#secrets)

Thanks in advance!

**Comment by zroubalik** [comment, marked as answer]
Hi, you should be able to specify a key in the TA's secret ref as well :)
